static/scss/partials/main.scss

```
@@ -379,7 +379,7 @@ button:active {
   opacity: 0.9;
 }
 
-.sign-in {
+.sign-in{
```

⚡ this specific space? 
```suggestion
.sign-in {
```